package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
-  "version": "1.1.3",
+  "version": "2.0.0",
```

Comment:
Breaking change since the `<MaintenanceBanner>` props are expecting `Date` objects instead of `moments`

Reply:
Ooh, good call. Let's be sure to make an announcement about this when we release it.